### PR TITLE
Add tests for model list table

### PR DESCRIPTION
**Pull request by hatched**
## Done

Add tests for the model list table and add mock data for testing with the redux store.

## QA

- CI should pass

### CODE REVIEW

File: src/testing/complete-redux-store-dump.js [hunk 1]
```diff
@@ -0,0 +1,2811 @@
...
+            instanceStatus: {
+              status: 'running',
+              info: 'RUNNING',
+              data: {},
+              since: '2017-03-09T20:40:24.303597554Z',
+              kind: '',
+              version: '',
+              life: ''
+            },
+            dnsName: '104.196.135.3',
```
Comment (ajbeattie) [inline src/testing/complete-redux-store-dump.js]: 104.196.135.3 happens to be a publicly visible Grafana :)

Comment (hatched) [reply src/testing/complete-redux-store-dump.js]: Hah, thanks!

File: src/testing/complete-redux-store-dump.js [hunk 2]
```diff
@@ -0,0 +1,2811 @@
...
+            instanceStatus: {
+              status: 'running',
+              info: 'RUNNING',
+              data: {},
+              since: '2016-12-12T17:36:03.774497893Z',
+              kind: '',
+              version: '',
+              life: ''
+            },
+            dnsName: '104.196.205.152',
```
Comment (ajbeattie) [inline src/testing/complete-redux-store-dump.js]: FWIW this seems to resolve but throws a 404 🤷‍♀

Comment (hatched) [reply src/testing/complete-redux-store-dump.js]: Hah, yeah I just randomly typed in numbers for the uuid's and ip's. I suppose there is always bound to be something real on the other end (especially for ipv4)

**Pull request merged**